recipe/meta.yaml

```
@@ -30,20 +30,18 @@ requirements:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - cython                                 # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
     - swig                                   # [build_platform != target_platform]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}  # [linux64 or ppc641le or win]
```

Think we can simplify this as well

```suggestion
    - {{ compiler('cuda') }}  # [(cuda_compiler or "None") != "None"]
```